### DISCUSSION TITLE
I am new to github and I don't know where/who to report this issue

### DISCUSSION THREAD

**Discussion by GakRop** [theOehrly/Fast-F1#161] [Q&A]
So I was browsing the data from 2022 season in fastf1 package, and found couple of data was missing. I found that the driver number of fastf1.api.car_data of Kevin Magnussen for Australian GP for FP1 and FP2 were mistakenly labeled as car number "9" (which is Nikita Mazepin). And the fastf1.api.car_data for Sebastian Vettel for Australian GP for FP1, FP2, FP3, Qualifying, and Sunday Race session data are missing. It is stored as list, not pandas dataframe.

Please let me know how to solve this issue or let me know who/where to report this.

**Comment by theOehrly** [comment]
You can create a new issue here: https://github.com/theOehrly/Fast-F1/issues

Preferably include a small self contained example to show the problem.

But, at least concerning the incorrect driver number, that sounds like an issue with the original data.
